validator/src/main.rs

```diff
@@ -25,60 +26,60 @@ pub fn main() {
-            commands::authorized_voter::execute(authorized_voter_subcommand_matches, &ledger_path);
+            commands::authorized_voter::execute(authorized_voter_subcommand_matches, &ledger_path)
         }
         ("plugin", Some(plugin_subcommand_matches)) => {
-            commands::plugin::execute(plugin_subcommand_matches, &ledger_path);
+            commands::plugin::execute(plugin_subcommand_matches, &ledger_path)
         }
         ("contact-info", Some(subcommand_matches)) => {
-            commands::contact_info::execute(subcommand_matches, &ledger_path);
+            commands::contact_info::execute(subcommand_matches, &ledger_path)
         }
         ("exit", Some(subcommand_matches)) => {
-            commands::exit::execute(subcommand_matches, &ledger_path);
-        }
-        ("monitor", _) => {
-            commands::monitor::execute(&matches, &ledger_path);
+            commands::exit::execute(subcommand_matches, &ledger_path)
         }
+        ("monitor", _) => commands::monitor::execute(&matches, &ledger_path),
         ("staked-nodes-overrides", Some(subcommand_matches)) => {
-            commands::staked_nodes_overrides::execute(subcommand_matches, &ledger_path);
+            commands::staked_nodes_overrides::execute(subcommand_matches, &ledger_path)
         }
         ("set-identity", Some(subcommand_matches)) => {
-            commands::set_identity::execute(subcommand_matches, &ledger_path);
+            commands::set_identity::execute(subcommand_matches, &ledger_path)
         }
         ("set-log-filter", Some(subcommand_matches)) => {
-            commands::set_log_filter::execute(subcommand_matches, &ledger_path);
+            commands::set_log_filter::execute(subcommand_matches, &ledger_path)
         }
         ("wait-for-restart-window", Some(subcommand_matches)) => {
-            commands::wait_for_restart_window::execute(subcommand_matches, &ledger_path);
+            commands::wait_for_restart_window::execute(subcommand_matches, &ledger_path)
         }
         ("repair-shred-from-peer", Some(subcommand_matches)) => {
-            commands::repair_shred_from_peer::execute(subcommand_matches, &ledger_path);
+            commands::repair_shred_from_peer::execute(subcommand_matches, &ledger_path)
         }
         ("repair-whitelist", Some(repair_whitelist_subcommand_matches)) => {
-            commands::repair_whitelist::execute(repair_whitelist_subcommand_matches, &ledger_path);
+            commands::repair_whitelist::execute(repair_whitelist_subcommand_matches, &ledger_path)
         }
         ("set-public-address", Some(subcommand_matches)) => {
-            commands::set_public_address::execute(subcommand_matches, &ledger_path);
+            commands::set_public_address::execute(subcommand_matches, &ledger_path)
         }
         _ => unreachable!(),
-    };
+    }
+    .unwrap_or_else(|err| {
+        println!("Validator command failed: {err}");
```

Comment:
One other note, I de-capitalized the first letter in each message in the individual command files since the caller here prepends `Validator command failed:` to the error message from underlying command. This is consistent with other areas of the codebase where we "chain" errors up the call-stack